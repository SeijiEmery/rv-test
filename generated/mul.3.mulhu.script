load /x 0 generated/mul.3.mulhu.hex

run 0 100000