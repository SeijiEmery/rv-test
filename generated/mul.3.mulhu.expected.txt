tests/mul.test.s:45

